hd/lang/lex_utf8.txt

```diff
@@ -1,3 +1,28 @@
+
+    delete/deleted
+en: delete/deleted
+fr: suppression des/supprimés
+
+    delete ancestors/deleted ancestors
+en: delete ancestors/deleted ancestors
+fr: suppression des ascendants/ascendants supprimés
+
+    confirm killing
```

Comment:
Maybe after each modification of lex_utf8, you should always run lex tool so that the file is always kept sorted and everything like that. Also while I'm in favor of new clear message, keep in mind that there is only 2 translations available.